umbraco/Umbraco-CMS
# Add config to limit concurrent logins

### Description
This adds new configuration that limits concurrent logins.
- `Umbraco:CMS:Security:AllowConcurrentLogins`

The value will be true for existing projects but false on new projects. We expect to change this so it defaults to false for v13, because this is breaking

### Tests
- `AllowConcurrentLogins == true`
  - Ensure you still concurrently can sign in from two different browsers
  
- `AllowConcurrentLogins == false`
  - Ensure you are signed out from the old browser, when signing in on a new (at most 30 sec wait)
